sdcm/docker.py

```diff
@@ -102,6 +96,32 @@ def destroy(self, force=True):  # pylint: disable=arguments-differ
         force_param = '-f' if force else ''
         _cmd('rm {} -v {}'.format(force_param, self.name))
 
+    def start_scylla_server(self, verify_up=True, verify_down=False, timeout=300):
```

Comment:
check again if we need to re-implement this

Reply:
seem like the current scylla-server.service doesn't work out of the box, I'll leave it to fallback to supervisor